app/oxalis/files/TempFileService.scala

```diff
@@ -0,0 +1,46 @@
+package oxalis.files
+
+import java.nio.file.{Files, Path, Paths}
+
+import com.scalableminds.util.tools.Fox
+import com.typesafe.scalalogging.LazyLogging
+import javax.inject.Inject
+import net.liftweb.common.Box.tryo
+import oxalis.cleanup.CleanUpService
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.{DurationInt, FiniteDuration}
+import scala.util.Random
+
+class TempFileService @Inject()(cleanUpService: CleanUpService)(implicit ec: ExecutionContext) extends LazyLogging {
+
+  private val tmpDir: Path = Paths.get(System.getProperty("java.io.tmpdir")).resolve("webKnossosTempFiles")
+
+  private val activeTempFiles = scala.collection.mutable.Set[(Path, Long)]()
+
+  cleanUpService.register("Clean up expired temporary files", 1 hour)(cleanUpExpiredFiles())
```

Comment:
Can we also enforce to clean all files (regardless of expiry) when the application is exiting? Assuming that `java.io.tmpdir` is cleaned up by the os on shutdown, files might still pile up when restarting the application, as non-expired files are still there.